src/render/svg.rs

```diff
@@ -51,7 +51,7 @@ impl<'a> RenderCanvas for Canvas<'a> {
                     r#"<svg xmlns="http://www.w3.org/2000/svg""#,
                     r#" version="1.1" width="{w}" height="{h}""#,
                     r#" viewBox="0 0 {w} {h}" shape-rendering="crispEdges">"#,
-                    r#"<rect x="0" y="0" width="{w}" height="{h}" fill="{bg}"/>"#,
+                    r#"<path d="M0 0h{w}v{h}h-{w}z" fill="{bg}"/>"#,
```

Comment:
```suggestion
                    r#"<path d="M0 0h{w}v{h}H0z" fill="{bg}"/>"#,
```

this should be even shorter? :thinking:

Reply:
Which is better `"M{left} {top}h{width}v{height}h-{width}z"` or `"M{left} {top}h{width}v{height}H{left}z"`?

Reply:
@sorairolake  if `left < width * 10` use `H{left}` otherwise `h-{width}`.

but always using `h-{width}` should compress better.